mcs/class/System.Data/mobile_System.Data.dll.sources

```diff
@@ -64,6 +65,6 @@ System.Data.Common/DbTypes.cs
 System.Data.Common/ExceptionHelper.cs
 System.Data.Common/DbConnectionStringBuilderHelper.cs
 
-#include mobile_referencesource.sources
-
+MobileStubs.cs
```

Comment:
It was in mobile_referencesource.sources before, and is not corefx specific: https://github.com/mono/mono/blob/master/mcs/class/System.Data/MobileStubs.cs